spark/v2.4/spark/src/main/java/org/apache/iceberg/spark/actions/BaseRewriteManifestsSparkAction.java

```diff
@@ -356,7 +356,7 @@ private static ManifestFile writeManifest(
         long snapshotId = row.getLong(0);
         long sequenceNumber = row.getLong(1);
         Row file = row.getStruct(2);
-        writer.existing(wrapper.wrap(file), snapshotId, sequenceNumber);
+        writer.existing(wrapper.wrap(file), snapshotId, sequenceNumber, null);
```

Comment:
@aokolnychyi would we need to backport https://github.com/apache/iceberg/pull/6193 to Spark 2.4 here?

Reply:
I've opened https://github.com/apache/iceberg/pull/6289 to do that

Reply:
Yep, thanks. I merged.